### ISSUE TITLE
Doesn't remember '≥' (greater-than or equal to)

### ISSUE THREAD

**Issue by cyberus-exe** [hluk/CopyQ#2242]
**Describe the bug**
It remembers everything fine, except for the single character '≥' (greater-than or equal to), that's never part of the copy history. Copying text that contains '≥' as well as other characters such as "a≥b" works fine, it's just when I copy '≥' by itself. Other characters (even similar ones) work fine, at least I haven't found any that show similar behavior.

**To Reproduce**
1. Copy ≥ from somewhere
2. Open CopyQ
3. Notice that ≥ is missing

**Expected behavior**
It should show ≥ in the history list

**Version, OS and Environment**
CopyQ Clipboard Manager 6.0.1
Qt: 5.15.2
KNotifications: 5.89.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: Ubuntu 22.04.1 LTS

**Comment by cyberus-exe** [comment]
I was using the Ubuntu (deb) version from the software center, which is on 6.0.1 with no updates available (you might want to update it there). I switched to the Flatpak version now (6.3.2) and it copies ≥ just fine.

So this is either an issue with the runtime (default Ubuntu runtime vs Flatpak runtime) or it has been fixed along the way (I can't find anything regarding that in the changelog).

New version is...
CopyQ Clipboard Manager 6.3.2
Qt: 5.15.7
KNotifications: 5.101.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: KDE Flatpak runtime
